### ISSUE TITLE
ADSK plugin fails to load - dev branch

### ISSUE THREAD

**Issue by ryan-seymour** [Autodesk/maya-usd#119]
I'm having trouble loading the mayaUsdPlugin.so plugin built from the dev branch with the following errors:

Script Editor
`// Error: Failed to register fragment 'UsdUVTexture' from file:  // `
Terminal
```
Coding Error: in _GetResourcePath at line 99 of /opt/build/dev/maya_usd/lib/render/vp2ShaderFragments/shaderFragments.cpp -- Failed verification: ' plugin ' -- Could not get plugin


API error detected in /opt/build/dev/maya_usd/lib/nodes/proxyShapePlugin.cpp at line 86
: (kFailure): Unexpected Internal Failure

API error detected in /opt/build/dev/maya_usd/plugin/adsk/plugin/plugin.cpp at line 48
: (kFailure): Unexpected Internal Failure
```
Trying to load the pxr or AL plugins yields the same error. 

It's entirely possible I've missed adding a path to an env var but I've tried adding various related paths to the MAYA_PLUG_IN_PATH, MAYA_SCRIPT_PATH and LD_LIBRARY_PATH with no success.

Just to note, I can successfully build and load the pxr/AL plugins from the master branch.

Building on CentOS-7.6.1810

Let me know if you need more info.

**Comment by santosg87** [comment]
Thanks for the information. I can see a similar behavior on my Linux machine as well. 

I am currently investigating it to see what could be the cause. we have this logged internally as MAYA-102025

**Comment by cfmoore007** [comment]
This error we did not see in the refactoring_sandbox, but do see in dev. 
Everything appears to be functioning properly w/ the plugin (unit tests and visual checks pass fine )
It appears that fragment shader .xml's are trying to get loaded from the pixar plugin logic (plugInfo.json) ?

**Comment by kxl-adsk** [comment]
The missing path for mayaUsdPlugin should be fixed by #118. FWIW. The first merge from refactoring sandbox was not supposed to bring this plugin but UFE tests relied on it. Currently, in the dev branch, this plugin only registers proxy shape, future merge from refactoring sandbox will bring Import/Export.

Errors reported during plugin load (as @cfmoore007 mentioned) don't affect functionality...but still have to be investigated and fixed.

**Comment by ryan-seymour** [comment]
Thanks for the responses.
After looking at #118 I realised that I should probably have just been adding the mayaUSD.mod.template to the MAYA_MODULE_PATH all along. I've been building the plugins since the 19.07 release and at that time, the mod template didn't exist so I was adding all the paths manually. 
I just tested this and it did resolve the original issue.

There is another small issue which I've had to work around though: 
If the python path of the usd installation comes before the python path of the pxr plugin in maya-usd in the sys.path, then UsdMaya will fail to import. This could be to do with how our packages are set up and, as I say, I have a workaround but just thought I would mention it in case it's something which should be addressed.

I'm also curious about compatibility with monolithic builds of usd as I'm yet to have any success compiling but I'll file this as a separate issue.

Thanks
